projects/replicated-supportbundle/spec-good.yaml

```diff
@@ -136,3 +136,33 @@ lifecycle:
 
 
 
+collect:
+  v1:
```

Comment:
I wish we had a good way to do multiple good spec files
That way we wouldn't need to have `collect.v1` and `specs` in the same document